### ISSUE TITLE
Gmail not working anymore

### ISSUE THREAD

**Issue by nb87** [k9mail/k-9#3476]
Please search to check for an existing issue (including closed issues, for which the fix may not have yet been released) before opening a new issue: https://github.com/k9mail/k-9/issues?q=is%3Aissue

### Expected behavior
Tell us what should happen
I should be able to send and receive emails (which i was until a few days ago)

### Actual behavior
Tell us what happens instead
No emails come in and when i try sending something i get a message that it cannot be sent. I try reentering my password and then i get a popup that k9 cannot connect to the server

### Steps to reproduce
[screenshot_2018-06-30-10-41-33](https://user-images.githubusercontent.com/40708759/42121192-891b0cf8-7c54-11e8-879d-fbc91cb8292e.png)

### Environment
K-9 Mail version: 5.403

Android version: 4.4.4

Account type (IMAP, POP3, WebDAV/Exchange): gmail, imap, smtp

**Comment by fake-fur** [comment]
that looks like a security block from google - i would do what it says and log in via a web browser to see if you can unblock the account / app

**Comment by Shaerif** [comment]
correct, but it's need to be more secure app..

It's actually bad if it's force you to allow less secure app to make k-9 to work.

**Comment by JohnScott623** [comment]
> It's actually bad if it's force you to allow less secure app to make k-9 to work.

That's not really true. By default, Gmail does not enable IMAP and SMTP, open protocols for sending and receiving mail. For the majority of Gmail users that use the webmail interface this is good, since Google's webmail supports signing in with two-factor authentication and other features the protocol does not support. It's good to leave them disabled if you only use webmail.

Enabling "less-secure apps" that use the open protocols is the only way for mail clients to send and receive mail: that's what the protocols were made for. Security isn't that black-and-white, and I think Google is to blame for their choice of words. It's less secure only if you don't need it.

This isn't a bug in K-9, though, since you'd have to complete the same steps to use Gmail in any other mail client, including Thunderbird.

**Comment by cketti** [comment]
See #655

**Comment by ArchangeGabriel** [comment]
>Enabling "less-secure apps" that use the open protocols is the only way for mail clients to send and receive mail: that's what the protocols were made for.

>This isn't a bug in K-9, though, since you'd have to complete the same steps to use Gmail in any other mail client, including Thunderbird.

@JohnScott623 : Both statements are not true. Thunderbird for instance use XOAUTH2, which is what #655 is about. And as I said in #3470, there is an alternative to using “less secure apps” for K-9 (as well as other apps).

**Comment by nb87** [comment]
Thanks for the advice but i did all that months ago, it worked fine and then one day it didnt anymore, without me changing anything. I already changed to a new app because i tried whatever i could find in this forum and nothing helped....
Cheers

-----Original Message-----

**Comment by sneakin** [comment]
The domain resolving to IPv6 was my problem. Had to manually resolve the IPv4 address and put that in K9.
